tools/generateTool.ps1

```diff
@@ -1,3 +1,16 @@
+# ----------------------------------------------------------------------------------
+    # Copyright Microsoft Corporation
+    # Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
@dsgouda we no longer use apache license.
We should be using MIT license.

Reply:
I simply copied what the powershell folks are doing, will make the changes.